101-webapp-privateendpoint-vnet-injection/azuredeploy.json

```diff
@@ -0,0 +1,395 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "virtualNetwork_name": {

```

Comment:
add a defaultValue

Reply:
I explicitly ask for the name of the vnet, added GEN in the parameter file

Reply:
Adding the defaultValue means the it's easier for the user to deploy, less margin for error and falls inline with other templates in the repo...  No reason to make the user enter this in the QuickStart scenario

Reply:
ok it's cristal clear, I understand, corrected